browser/about_flags.cc

```diff
@@ -307,6 +311,13 @@ constexpr char kNavigatorConnectionAttributeDescription[] =
 #define BRAVE_VPN_FEATURE_ENTRIES
 #endif
 
+#define BRAVE_SKU_SDK_FEATURE_ENTRIES                  \
+    {"skus-sdk",                                       \
+     flag_descriptions::kBraveSkusSdkName,             \
+     flag_descriptions::kBraveSkusSdkDescription,      \
+     kOsMac | kOsWin,                                  \
```

Comment:
why do we have OS specified here?

Reply:
Needs to be limited to macOS and Windows; although I missed that we need to Android in here too

For iOS, SKU SDK should ALWAYS be enabled and doesn't need a flag

Reply:
skus does or vpn? Also ios does need a flag because the feature has a flag

Reply:
OK will add iOS also

Reply:
Does iOS have a way to change feature flags? Not sure if I should add this in if there's not a way to change it

Reply:
@bsclifton iOS doesn't have any way of altering feature flags at this time, we usually setup the `FeatureFlag` with `#ifdef`'s to enable by default

Reply:
I think we're good excluding iOS; as @kylehickinson suggested below, I did a commit to enable by default on iOS:
https://github.com/brave/brave-core/pull/10847/commits/2d6ef57d9ff97f291e0975858df498e32973c7b4